### ISSUE TITLE
Error overlay for runtime errors

### ISSUE THREAD

**Issue by KutnerUri** [webpack/webpack-dev-server#4308]
### Feature Proposal

I want to use the built in error overlay to handle runtime errors.
Is this possible? what is the configuration to do that?

Also, I see create-react-app switched to using the built-in error overlay. Does this mean it converged with the `react-error-overlay` like in #3689 ?
```ts
plugins: [
   // ...
  new ReactRefreshWebpackPlugin({
    overlay: false,
  }),
]
```
```ts
client: {
  // ...
  overlay: {
    errors: true,
    warnings: false,
  },
},
```

### Feature Use Case

I am making a custom workspace tool - http://bit.dev/, similar to create-react-app, and I want the same error handling experience.

### Please paste the results of `npx webpack-cli info` here, and mention other relevant information

```
❯ npx webpack-cli info

  System:
    OS: macOS 12.2.1
    CPU: (16) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
    Memory: 1.78 GB / 32.00 GB
  Binaries:
    Node: 12.22.7 - ~/.nvm/versions/node/v12.22.7/bin/node
    Yarn: 1.22.17 - ~/.nvm/versions/node/v12.22.7/bin/yarn
    npm: 6.14.15 - ~/.nvm/versions/node/v12.22.7/bin/npm
  Browsers:
    Chrome: 98.0.4758.102
    Edge: 98.0.1108.56
    Firefox: 97.0
    Safari: 15.3
  Packages:
    babel-loader: ^8.2.3 => 8.2.3 
    case-sensitive-paths-webpack-plugin: ^2.4.0 => 2.4.0 
    css-loader: ^6.5.1 => 6.6.0 
    css-minimizer-webpack-plugin: ^3.2.0 => 3.4.1 
    eslint-webpack-plugin: ^3.1.1 => 3.1.1 
    file-loader: ^6.2.0 => 6.2.0 
    html-webpack-plugin: ^5.5.0 => 5.5.0 
    postcss-loader: ^6.2.1 => 6.2.1 
    resolve-url-loader: ^4.0.0 => 4.0.0 
    sass-loader: ^12.3.0 => 12.6.0 
    source-map-loader: ^3.0.0 => 3.0.1 
    style-loader: ^3.3.1 => 3.3.1 
    terser-webpack-plugin: ^5.2.5 => 5.3.1 
    webpack: ^5.64.4 => 5.69.1 
    webpack-dev-server: ^4.6.0 => 4.7.4 
    webpack-manifest-plugin: ^4.0.2 => 4.1.1 
    workbox-webpack-plugin: ^6.4.1 => 6.4.2 
  Global Packages:
    webpack-cli: 4.9.1
```

**Comment by alexander-akait** [comment]
PR welcome, duplicate of https://github.com/webpack/webpack-dev-server/issues/3689

**Comment by RagulTW** [comment]
It is now implemented in webpack dev sever version 4.8.1
